addons/hr/wizard/hr_plan_wizard_views.xml

```diff
@@ -19,7 +19,7 @@
                     </sheet>
                     <footer>
                         <button name="action_launch" string="Launch Plan" type="object" class="oe_highlight" attrs="{'invisible': [('warning', '!=', False)]}" groups="hr.group_hr_user" data-hotkey="q"/>
-                        <button name="action_launch" string="Launch Plan" type="object" class="oe_highlight" disabled="disabled" attrs="{'invisible': [('warning', '=', False)]}" groups="hr.group_hr_user" data-hotkey="q"/>
+                        <button string="Launch Plan" class="oe_highlight" disabled="disabled" attrs="{'invisible': [('warning', '=', False)]}" groups="hr.group_hr_user" data-hotkey="q"/>
```

Comment:
```suggestion
                        <button name="action_launch" string="Launch Plan" type="object" class="oe_highlight disabled" attrs="{'invisible': [('warning', '=', False)]}" groups="hr.group_hr_user" data-hotkey="q"/>
```

Reply:
Hello @niyasraphy, thanks for the fix :innocent: 

What do you think if we do like I suggested?

Reply:
hi @sofiagvaladze updated the pr as per the suggestion :)